console/src/components/Services/Data/RawSQL/molecules/NotesSection.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+/**
+ * # NoteSection React static component
+ *
+ * @returns
+ */
+const NotesSection = () => {
```

Comment:
We want new files to be created in TypeScript. Could you change it to `.tsx` file?

Reply:
Sure Ill change it